proposals/0306-actors.md

```diff
@@ -136,7 +142,7 @@ extension BankAccount {
 }
 ```
 
-The `deposit(amount:)` operation needs to involve the state of a different actor, so it must be invoked asynchronously. This method could itself be implemented as `async`:
+The `deposit(amount:)` operation needs involve the state of a different actor, so it must be invoked asynchronously. This method could itself be implemented as `async`:
```

Comment:
This change seems to be replacing contemporary usage with that of a regency novel